EisenhowerTaskManager/EisenhowerTaskManager/Views/Matrix/TaskTableViewCell/TaskTableViewCell.swift

```diff
@@ -41,9 +41,9 @@ extension TaskTableViewCell {
             lastEditDateLabel?.text =
                 L10n.Generic.today +
                 " " +
-                (task?.lastEditDate.string(dateStyle: .none, timeStyle: .short) ?? "")
+                (task?.lastEditDate.toString(DateToStringStyles.time(DateFormatter.Style.short)) ?? "")
         } else {
-            lastEditDateLabel?.text = task?.lastEditDate.string(dateStyle: .short, timeStyle: .short)
+            lastEditDateLabel?.text = task?.lastEditDate.toString(DateToStringStyles.dateTime(DateFormatter.Style.short))
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gorbat-o/EisenhowerTaskManager/blob/fb610e11359f11b5c379a47437dd97a72e58e9d8/EisenhowerTaskManager/EisenhowerTaskManager/Views/Matrix/TaskTableViewCell/TaskTableViewCell.swift#L46"></span>Line should be 120 characters or less: currently 121 characters</td>
    </tr>
  </tbody>
</table>